webpack.config.js

```diff
@@ -26,4 +25,18 @@ module.exports = {
     ],
   },
   devtool: 'source-map',
+  optimization: {
```

Comment:
By default, the TerserPlugin in used by webpack 4 for optimization. This webpack configuration is pretty simple. Is there a reason we can't use the default?

Reply:
This is just copy-pasted from terra-toolkit: https://github.com/cerner/terra-toolkit/blob/master/config/webpack/webpack.config.js#L125-L138